npm security updates

Samua11y opened the pull request:
Add ignore-scripts to npmrc
Add save-exact to npmrc
Update installation instructions in README
Remove version ranges from package.json
Update package-lock.json

Samua11y committed:
npm security updates

sean-m-carroll commented on .npmrc:
```diff
@@ -0,0 +1,2 @@
+ignore-scripts=true
```
It might be worth adding `save-prefix=""` too.

Are you adding any guidance around installing new dependencies, or that updates must be 7 days or older before using them?

Samua11y replied:
Have added `save-prefix=""` thanks.

No specific guidance for now - can add that once we have decided upon wording.

Samua11y committed:
Addition of save-prefix to npmrc

Samua11y committed:
Version updates

Specific version in nvmrc
Update dependency versions

Samua11y committed:
Remove preinstall script and revert package-lock version

Remove the preinstall script and related resolutions from package.json.

Return package-lock.json to lockfileVersion 1.

randombitsUK reviewed (approved):
LGTM
